src/routes.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import { SideBar } from './sections/SideBar/';
+import { Home } from './sections/Home/';
+import { AboutMe } from './sections/AboutMe/';
+import { Resume } from './sections/Resume/';
+import { ContactMe } from './sections/ContactMe/';
+import { NotFound } from './sections/NotFound/';
```

Comment:
Please create an index file at /section level and then export from that file all those sections so that you can then do the following:

`import { NotFound, SideBar, Home , ...  } from './sections';`